Examples/camera_360_test/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../../common/flags.mk
+CXXFLAGS += -I../..
+
+.PHONY: all clean
+
+all: camera_360_test
+
+camera_360_test:
```

Comment:
Shouldn't this depend on the .cc and the .d?

Reply:
Good spot.